# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/qt6webengine/hotfix-gn-deps.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ./src/3rdparty/chromium/chrome/test/chromedriver/BUILD.gn.vanilla	2024-04-25 21:14:51.985000000 +0200
+++ ./src/3rdparty/chromium/chrome/test/chromedriver/BUILD.gn	2024-04-25 21:23:06.107000000 +0200
@@ -432,6 +432,7 @@
       ":automation_client_lib",
       ":lib",
       "//build:chromeos_buildflags",
+      "//ui/gl:buildflags",
       "//build/win:default_exe_manifest",
       "//mojo/core/embedder",
       "//net/server:http_server",

--- ./src/3rdparty/chromium/content/browser/BUILD.gn.orig        2023-02-27 18:57:08.000000000 +0000
+++ ./src/3rdparty/chromium/content/browser/BUILD.gn
@@ -36,5 +36,7 @@
     "//base",
+    "//components/spellcheck:buildflags",
+    "//ui/gl:buildflags",
     "//content/browser/devtools:devtools_background_services_proto",
     "//content/browser/devtools:protocol_sources",
     "//content/common:buildflags",

--- a/src/3rdparty/chromium/extensions/browser/api/declarative_net_request/BUILD.gn
+++ b/src/3rdparty/chromium/extensions/browser/api/declarative_net_request/BUILD.gn
@@ -22,2 +22,3 @@
     "//content/public/browser",
+    "//components/web_cache/public/mojom",
     "//extensions/common",
--- qtwebengine-everywhere-src-6.8.2/src/core/configure/BUILD.root.gn.in.vanilla	2025-02-04 18:44:59.069999964 +0100
+++ qtwebengine-everywhere-src-6.8.2/src/core/configure/BUILD.root.gn.in	2025-02-04 18:45:39.299999967 +0100
@@ -237,7 +237,6 @@
 source_set("devtools_sources") {
   configs += [ ":cpp20_config" ]
   deps = [
-    "//chrome/app:generated_resources",
     "//components/zoom",
     "//third_party/blink/public/mojom:mojom_platform",
     ]

